frontend/src/lib/lemon-ui/LemonButton/LemonButton.scss

```diff
@@ -220,6 +220,16 @@
         --lemon-button-icon-opacity: 1;
     }
 
+    &.LemonButton--primary.LemonButton--status-danger {
+        --lemon-button-bg-color: var(--red-100);
+        --lemon-button-color: var(--text-3000);
+
+        [theme='dark'] & {
+            --lemon-button-bg-color: #312101;
```

Comment:
Feels weird to use a one-off color here, although I haven't given it thought how to fit it into the system